Makefile

```diff
@@ -6,7 +6,7 @@ KUBECTLCMD=$(shell which kubectl)
 SWAGGER := $(DOCKERCMD) run --rm -it -v $(HOME):$(HOME) -w $(shell pwd) quay.io/goswagger/swagger
 
 REGISTRY ?= projects.registry.vmware.com/cnsi
-IMG_TAG = 0.3
+IMG_TAG = dev
```

Comment:
Just for clarification: Is this tag only for testing purpose?

Reply:
Yes. I should update it with the latest tag. Done